File: app/src/main/java/com/tribyssapps/refreshmentinvestment/SplashScreenActivity.java
package com.tribyssapps.refreshmentinvestment;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.tribyssapps.refreshmentinvestment.activities.MainActivity;

public class SplashScreenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        startActivity(new Intent(this, MainActivity.class));
        finishAffinity();
    }
}

File: app/src/main/java/com/tribyssapps/refreshmentinvestment/SplashScreenActivity.kt
package com.tribyssapps.refreshmentinvestment

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.tribyssapps.refreshmentinvestment.activities.MainActivity

class SplashScreenActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        startActivity(Intent(this, MainActivity::class.java))
        finishAffinity()
    }
}
